roles/edpm_ceph_hci_pre/defaults/main.yml

```diff
@@ -0,0 +1,122 @@
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+
+# All variables within this role should have a prefix of "edpm_ceph_hci_pre"
+
+# Firewall configuration
+edpm_ceph_hci_pre_configure_firewall: true
+
+# Only allow default storage network to access Ceph RBD/Mon
+edpm_ceph_hci_pre_storage_ranges:
+  - 172.18.0.0/24
+
+# What list of IP ranges should connect to RGW front end?
+# If the list is empty, no RGW frontend ports will be opened
+# append 0.0.0.0/0 to allow access from all IPs
+edpm_ceph_hci_pre_rgw_frontend_ranges: []
+
+# What list of IP ranges should connect to Ceph Grafana front end?
+# If the list is empty, no Grafana frontend ports will be opened
+edpm_ceph_hci_pre_grafana_frontend_ranges: []
+
+# What list of IP ranges should connect to RBD Mirror?
+# If the list is empty, no RBD Mirror ports will be opened
+edpm_ceph_hci_pre_rbd_mirror_ranges: []
+
+edpm_ceph_hci_pre_firewall_services:
+  - name: ceph_mon
+    num: 110
+    dport:
+      - 6789
+      - 3300
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+  - name: ceph_osd
+    num: 111
+    dport:
+      - 6800:7300
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+  - name: ceph_mgr
+    num: 113
+    dport:
+      - 6800:7300
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+  - name: ceph_mds
+    num: 112
+    dport:
+      - 6800:7300
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+  - name: ceph_nfs
+    num: 120
+    dport:
+      - 2049
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+  - name: ceph_rgw
+    num: 122
+    dport:
+      - 8080
+      - 13808
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+  - name: ceph_rgw_frontend
+    num: 100
+    dport:
+      - 8080
+    ranges: "{{ edpm_ceph_hci_pre_rgw_frontend_ranges | list }}"
+  - name: ceph_ssl_rgw_frontend
+    num: 100
+    dport:
+      - 13808
+    ranges: "{{ edpm_ceph_hci_pre_rgw_frontend_ranges | list }}"
+  - name: ceph_rbdmirror
+    num: 114
+    dport:
+      - 6800:7300
+    ranges: "{{ edpm_ceph_hci_pre_rbd_mirror_ranges | list }}"
+  - name: ceph_grafana
+    num: 123
+    dport:
+      - 3100
+      - 9100
+      - 9090
+      - 9092
+      - 9093
+      - 9094
+      - 9100
+      - 9283
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+  - name: ceph_grafana_frontend
+    num: 100
+    dport:
+      - 3100
+    ranges: "{{ edpm_ceph_hci_pre_grafana_frontend_ranges | list }}"
+  - name: ceph_prometheus
+    num: 100
+    dport:
+      - 9092
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+  - name: ceph_alertmanager
+    num: 100
+    dport:
+      - 9093
+    ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
+
```

Comment:
``` 
- name: ceph_dashboard
  num: 100
  dport:
      - 8444
  ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
```

Reply:
or, we can reuse `ceph_grafana` and add 8444 there, but technically it's a diff svc

Reply:
OK, I added `ceph_dashboard`. Thanks for catching that.